### DISCUSSION TITLE
What's one thing that you wish your BI tool could do that it doesn't do right now?

### DISCUSSION THREAD

**Discussion by TuringLovesDeathMetal** [lightdash/lightdash#1099] [General]
https://media1.giphy.com/media/H7kfFDvD9HSYGRbvid/200w.gif?cid=82a1493bzsjtxkm17rrpiczmc6xg1qm0oa5c2li1hg2v2sj5&rid=200w.gif&ct=g

Sometimes it's nice to imagine a place where your data lake is so perfect you just want to swim in it, your warehouse is organized so well, Mary Kondo would be proud, and your BI tool answers everyone's data questions before they even ask them.

So, in the spirit of "happy places", constructive complaining, and feeding our product ideation backlog at Lightdash (:heart:) :

### What's one thing that you wish your BI tool could do that it doesn't do right now?

**Comment by TuringLovesDeathMetal** [comment]
## ....I'll go first:

> I wish my BI tool gave me relevant queries or plots when I'm trying to explore my data 

Imagine,  opening up a BI tool, clicking on a table you want to explore, and the empty-state (before you've clicked on anything) is filled with suggested queries/charts for you:
<img src="https://user-images.githubusercontent.com/31848148/148254881-234c6f0f-1588-46c2-b445-e9c09a497793.png" alt="drawing" width="600"/>

And having this similar "relevant/popular data" for a user on their homepage, when they open the app. This could be based on something like their product squad or role (what are other users in their product squad/role looking at?).

Okay, sure, so you're able to curate your own BI tools using folders and homepages, etc. But a business and its data are *dynamic*, so trying to keep a perfectly curated, relevant set of dashboards/explores for everyone is just chaos (*shudders as she relives her experiences of this... *)

All of the BI tools I've used don't give you context when you're trying to explore data. This makes it really intimidating to open up and really dig into the BI tool for folks who aren't using this all day, ever day. Also, if you're a data person looking at solving a problem related to another product/squad, then just having the most relevant dashboards available to you immediately makes your job that much easier.

Giving someone enough context/a foundation to build from might give them the little step up they needed to get the answer they were looking for (which would be awesome!).

**Reply by jaroet** [reply]
In the past we worked with YellowFin and it did just that ... show you what other people where looking for and with the explanation option it self explains the reports you are looking to with self-generatated text's and charts. 

Another tool doing that extremely well (not available anymore unfortunately) was DynamicAI. Their website is still up and the video's at the bottom-right show features and functionality unrivalled to this day although the look and feel is starting to show it's age.

**Comment by djayatillake** [comment]
Automatically delete unused dashboards and looks and then any code that only is used by these. If it doesn't bring you joy...

**Comment by Nopalin** [comment]
Charts are no the only thing users need to make desicions, the truth is that charts are made for presentations in a meeting, but a lot of users need to extract all kind of data and transform to analize it, and this mostly happen in excel. Charts are to meetings as Excel are to ops users. So it would be wonderfull if in lightdash one could design reports to just visualize them in tables and export them to excel (no CVS no JSON but in EXCEL). I have tested a few BI's but none of them can do just that, all of them generate data to visualize in charts.
